scripts/smoke_tests/setup.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+import sys
+import json
+import time
+import threading
+
+import requests
+
+from tests.functional.backend.common import BaseFunctionalTestCase
+
+# min amount of published collections expected by smoke tests
+MIN_TEST_COLLECTIONS = 4
+# amount advised by Timmy Huang to reduce flakiness
+MAX_TEST_COLLECTIONS = 10
+
+
+class SmokeTestsInitializer(BaseFunctionalTestCase):
+
+    def __init__(self):
+        super().setUpClass()
+        super().setUp()
+        self.headers = {"Cookie": f"cxguser={self.curator_cookie}", "Content-Type": "application/json"}
+
+    def get_collection_count(self):
+        res = self.session.get(f"{self.api}/curation/v1/collections?visiblity=PUBLIC", headers=self.headers)
+        res.raise_for_status()
+        data = json.loads(res.content)
+        num_collections = 0
+        for collection in data['collections']:
+            if collection["contact_name"] == "Smoke Test User":
+                num_collections += 1
+            if num_collections >= MIN_TEST_COLLECTIONS:
```

Comment:
num_collections will never be > than MIN_TEST_COLLECTIONS right? Because you're incrementing by 1 and as soon as you hit MIN_TEST_COLLECTIONS, then it returns so the max it can ever be is MIN_TEST_COLLECTIONS. 

If my logic is correct, I would simplify to `if num_collections == MIN_TEST_COLLECTIONS:`